src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -773,269 +774,292 @@ static HtmlEntities()
-                    ["rsaquo"] = '\x203a',
-                    ["oline"] = '\x203e',
-                    ["frasl"] = '\x2044',
-                    ["euro"] = '\x20ac',
-                    ["image"] = '\x2111',
-                    ["weierp"] = '\x2118',
-                    ["real"] = '\x211c',
-                    ["trade"] = '\x2122',
-                    ["alefsym"] = '\x2135',
-                    ["larr"] = '\x2190',
-                    ["uarr"] = '\x2191',
-                    ["rarr"] = '\x2192',
-                    ["darr"] = '\x2193',
-                    ["harr"] = '\x2194',
-                    ["crarr"] = '\x21b5',
-                    ["lArr"] = '\x21d0',
-                    ["uArr"] = '\x21d1',
-                    ["rArr"] = '\x21d2',
-                    ["dArr"] = '\x21d3',
-                    ["hArr"] = '\x21d4',
-                    ["forall"] = '\x2200',
-                    ["part"] = '\x2202',
-                    ["exist"] = '\x2203',
-                    ["empty"] = '\x2205',
-                    ["nabla"] = '\x2207',
-                    ["isin"] = '\x2208',
-                    ["notin"] = '\x2209',
-                    ["ni"] = '\x220b',
-                    ["prod"] = '\x220f',
-                    ["sum"] = '\x2211',
-                    ["minus"] = '\x2212',
-                    ["lowast"] = '\x2217',
-                    ["radic"] = '\x221a',
-                    ["prop"] = '\x221d',
-                    ["infin"] = '\x221e',
-                    ["ang"] = '\x2220',
-                    ["and"] = '\x2227',
-                    ["or"] = '\x2228',
-                    ["cap"] = '\x2229',
-                    ["cup"] = '\x222a',
-                    ["int"] = '\x222b',
-                    ["there4"] = '\x2234',
-                    ["sim"] = '\x223c',
-                    ["cong"] = '\x2245',
-                    ["asymp"] = '\x2248',
-                    ["ne"] = '\x2260',
-                    ["equiv"] = '\x2261',
-                    ["le"] = '\x2264',
-                    ["ge"] = '\x2265',
-                    ["sub"] = '\x2282',
-                    ["sup"] = '\x2283',
-                    ["nsub"] = '\x2284',
-                    ["sube"] = '\x2286',
-                    ["supe"] = '\x2287',
-                    ["oplus"] = '\x2295',
-                    ["otimes"] = '\x2297',
-                    ["perp"] = '\x22a5',
-                    ["sdot"] = '\x22c5',
-                    ["lceil"] = '\x2308',
-                    ["rceil"] = '\x2309',
-                    ["lfloor"] = '\x230a',
-                    ["rfloor"] = '\x230b',
-                    ["lang"] = '\x2329',
-                    ["rang"] = '\x232a',
-                    ["loz"] = '\x25ca',
-                    ["spades"] = '\x2660',
-                    ["clubs"] = '\x2663',
-                    ["hearts"] = '\x2665',
-                    ["diams"] = '\x2666',
+                    [1953461617] = '\x0022', // "quot"
```

Comment:
This will be very hard to maintain. Since this table is computed once, I'd rather see a helper added that converts strong to ulong, and then change each entry to just use that helper, e.g.
```c#
 [ToUInt64Key("quot")] = '\x0022',
```
You can then use the same helper in the lookup.